taskt/Core/Automation/Commands/Dictionary Commands/CreateDictionaryCommand.cs

```diff
@@ -1,94 +1,74 @@
 ﻿using System;
-using System.Xml.Serialization;
-using System.Data;
 using System.Collections.Generic;
-using System.Windows.Forms;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
-using System.Drawing;
+using System.Data;
 using System.Linq;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Dictionary Commands")]
-    [Attributes.ClassAttributes.Description("This command created a DataTable with the column names provided")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to create a new Dictionary")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Dictionary Commands")]
+    [Description("This command creates a new Dictionary.")]
     public class CreateDictionaryCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Indicate Dictionary Name")]
-        [Attributes.PropertyAttributes.InputSpecification("Indicate a unique reference name for later use")]
-        [Attributes.PropertyAttributes.SampleUsage("vMyDictionary")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Dictionary")]
+        [InputSpecification("Specify a unique reference name for new dictionary.")]
```

Comment:
might want to change to "for the new dictionary"

Reply:
done